File: URI/Java/IdadeEmDias.java
package Java;

import java.util.Scanner;

public class IdadeEmDias {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int time = input.nextInt();
        
        System.out.println(time/365 + " ano(s)");
        time %= 365;
        System.out.println(time/30 + " mes(es)");
        time %= 30;
        System.out.println(time + " dia(s)");

        input.close();
    }
}

File: URI/Python/IdadeEmDias.py
tmp = int(input())
anos = tmp/365
tmp = tmp%365
meses = tmp/30
tmp = tmp%30
dias = tmp
print("{} ano(s)".format(int(anos)))
print("{} mes(es)".format(int(meses)))
print("{} dia(s)".format(int(dias)))